web/src/components/apps/AppConfig.jsx

```diff
@@ -176,7 +176,8 @@ class AppConfig extends Component {
     if (match.params.sequence != undefined) {
       return parseInt(match.params.sequence);
     }
-    return app?.currentSequence;
+
+    return app?.downstreams[0]?.currentVersion?.parentSequence || app?.currentSequence;
```

Comment:
what if the `parentSequence` is `0`? like the currently deployed version is sequence `0` :)